### PR TITLE
created second demo branch

### PR DESCRIPTION
**Pull request by tiongsonkj**
creating second pull request

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Kelvin</title>
```
Comment (paulcruse3) [inline index.html]: Dont like this title

File: index.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+</head>
+<body>
+    <div>Kelvin's branch</div>
+</body>
+
+<script src="./src/index.js"></script>
```
Comment (paulcruse3) [inline index.html]: need to src from new location

File: index.html (hunk 1 above)
Comment (tiongsonkj) [reply index.html]: will change

File: index.html (hunk 2 above)
Comment (tiongsonkj) [reply index.html]: resolved

**Pull request merged**